tests/ext/test_jobqueue.py

```diff
@@ -85,6 +85,22 @@ class TestJobQueue:
         " We recommend double checking if the passed value is correct."
     )
 
+    async def test_repr(self, app):
+        import datetime
+
+        jq = JobQueue()
+        jq.set_application(app)
+        assert repr(jq) == f"JobQueue[application={app!r}]"
+
+        callback = self.job_run_once
+        job = jq.run_once(callback, 0, name="name2")
+        assert repr(job) == (
+            f"Job[id={job.job.id}, name={job.name}, callback=job_run_once, "
+            f"trigger=date["
+            f"{datetime.datetime.now(tz=datetime.timezone.utc).strftime('%Y-%m-%d %H:%M:%S UTC')}"
```

Comment:
this looks like it can be flaky. how about defining a fixed date in the future and using that in both places? i.e.

```python
dtm = datetime.utcnow() + timedelta(year=1)
job = jq.run_once(callback, dtm)
assert repr(job) == (… dtm.strftime(…))
```

Reply:
I thought `0` would eliminate the flakiness 😵‍💫 
Done :)